### ISSUE TITLE
05) Create Your Theme from Your Template.

### ISSUE THREAD

**Issue by msinkula** [bankie606/web170#1]
Make sure you upload your code so I can mark you off for 05) Create Your Theme from Your Template.

**Comment by bankie606** [comment]
I just uploaded the update code.

**Comment by msinkula** [comment]
I'm not seeing the assignment requirements in index.php?

**Comment by bankie606** [comment]
Yes. I've been having trouble implementing some of them. Wordpress blew up when I added some functions to my function file, and while I've sort of gotten things back to normal, I haven't added all the requirements yet.

**Comment by msinkula** [comment]
Just let me know when it is done.

**Comment by bankie606** [comment]
I think it's done now, sorry about that.
